models/user.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const userSchema = new Schema({
+  firstName: { type: String, required: true },
+  lastName: { type: String, required: true },
+  phone: Number,
+  email: {type: String, unique: true, required: true, },
+  password: {type: String, required: true, minlenght: [6, 'Password is to short']},
+  adress: String,
```

Comment:
address is missing a letter d on line 10
```suggestion
  address: String,
```